code-challenges.js

```diff
@@ -13,52 +13,139 @@ var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
 
+//Creating a function that will take in a number and attempt to divide it by three and return a pleasant little string along with it. 
+const divisibleBy = (num) =>{
+    if(num / 3){
+        return `${num} is divisible by three!`
+    } else { return `${num} is not divisible by three  :( `
+    }
+}
+
+//Logging the function call three times, with the set variables above as arguments. I know there is a better way of doing this, but just wanted to do the long way
+console.log(divisibleBy(num1))
+console.log(divisibleBy(num2))
+console.log(divisibleBy(num3))
+*/
+
+
 
 
 
 // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
+/*
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
+//Creating a function that will take in the array and spit out the array with only the first letter capitalized. 
+const capitalizedNouns = (nouns) => {
+    for (const nouns of randomNouns ){
+        console.log(nouns[0].toUpperCase() + nouns.substr(1))
+    }
+}
+
+//Calling the function to execute
+capitalizedNouns()
```

Comment:
this works but there's a tiny change you would need to make in order for this to be reusable